pkg/descheduler/descheduler.go

```diff
@@ -83,6 +86,28 @@ func Run(ctx context.Context, rs *options.DeschedulerServer) error {
 		return fmt.Errorf("deschedulerPolicy is nil")
 	}
 
+	// Add k8s compatibility warnings to logs
+	serverVersion, serverErr := rs.Client.Discovery().ServerVersion()
+	if serverErr != nil {
+		klog.V(1).InfoS("Warning: Get Kubernetes server version fail")
+		return serverErr
+	}
+
+	deschedulerMinorVersion := strings.Split(version.Get().Minor, ".")[0]
+	deschedulerMinorVersionInt, err := strconv.Atoi(deschedulerMinorVersion)
+	if err != nil {
+		klog.Warning("Warning: Convert Kubernetes server minor version to int fail")
+	}
+	kubernetesMinorVersionInt, err := strconv.Atoi(serverVersion.Minor)
+	if err != nil {
+		klog.Warning("Warning: Convert Kubernetes server minor version to int fail")
+	}
+	if deschedulerMinorVersion == serverVersion.Minor || deschedulerMinorVersionInt-kubernetesMinorVersionInt < 3 {
```

Comment:
```suggestion
	if deschedulerMinorVersionInt-kubernetesMinorVersionInt < 3 {
```
don't need to check if they're equal if we're also checking that their diff is < 3 :)

Reply:
actually let's change this to check `if > 3` and drop the V(1) message about descheduler being supported on your version. we only need to log a message/warning if the descheduler version is not supported

Reply:
sure, commit as your suggestion~

Reply:
may want to make that `Abs`

Descheduler 0.21 is not supported on Kubernetes 1.17 or 1.25

Reply:
thanks a lot~~ good finding~. i will update now

Reply:
update done